CHANGES.rst

```diff
@@ -392,6 +403,11 @@ astropy.constants
 
 astropy.convolution
 ^^^^^^^^^^^^^^^^^^^
+  - When writing to HDF5 files, the serialized metadata are now saved in a new
```

Comment:
the changelog seems to be a bit messed up after the rebase, could you clean it up a bit and also add the PR number to the end of the entry?